cvmfs/fd_refcount_mgr.h

```diff
@@ -9,7 +9,7 @@
 
 #include "cache.h"
 #include "smallhash.h"
-
+#include <gtest/gtest_prod.h>
```

Comment:
```suggestion
#include "duplex_testing.h"
```
See https://github.com/cvmfs/cvmfs/pull/3949